### DISCUSSION TITLE
allWpCategory count in wpgraphql is not updating when I start assigning post in the categories.

### DISCUSSION THREAD

**Discussion by sarniel** [wp-graphql/wp-graphql#2829] [GatsbyJS]
Hi,

I hope everyone see this and have the same issue. 

I'm using headless approach with gatsbyjs, my problem is the count, IT'S not updating when I start assigning post in the categories. It's still showing null, even though there is a post in the category. I understand, running yarn clean will fix everything, but shouldn't be automatically updating the graphql as will as in the gatsby cloud?

```
  allWpCategory(filter: {slug: {eq: "boxing"}}) {
    nodes {
      name
      count
    }
  }
```

I'm expecting to see value in the category count without clearing cache.

**Comment by justlevine** [comment]
@sarniel this sounds like an issue (or intended behavior) with the WPGatsby plugin, which is responsible for telling Gatsby Cloud when a WP change warrants an update. I suggest you ask in [that repo](https://github.com/gatsbyjs/wp-gatsby).

**Reply by sarniel** [reply]
Hi @justlevine, thank you for your comment. I just posted a question in their repo, I hope it makes sense to them.
